### PR TITLE
Update microsoft-365-multi-geo.md

### LINKED ISSUE #12824: Licenses must be assigned
The article says 
> Similarly, CSP partners must purchase a quantity of Multi-Geo licenses equal to or greater than 5% of their customer's total eligible Microsoft 365 seats.

However, in addition to being purchased, the licenses must be assigned before multi-geo features light up.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 32b41c7e-29cd-eb5b-36b2-738b3babd08f
* Version Independent ID: 22eda3f7-1c04-b3b0-95fd-3474b3f991b2
* Content: [Microsoft 365 Multi-Geo - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/microsoft-365-multi-geo?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/microsoft-365-multi-geo.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/microsoft-365-multi-geo.md)
* Service: **microsoft-365-enterprise**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

### PR DESCRIPTION
**Pull request by msbemba**
Added statement about assigning the license and not just purchasing

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12824

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/microsoft-365-multi-geo.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ In a Multi-Geo environment, your Microsoft 365 _Tenant_ consists of a central lo
 
 Microsoft 365 Multi-Geo is available as an add-on to the following Microsoft 365 subscription plans.
 
-Enterprise Agreement customers must purchase a quantity of Multi-Geo licenses equal to or greater than 5% of their total eligible seats. Similarly, CSP partners must purchase a quantity of Multi-Geo licenses equal to or greater than 5% of their customer's total eligible Microsoft 365 seats. For Enterprise customers, user subscription licenses must be on the same Enterprise Agreement as the Multi-Geo Services licenses. Contact your Microsoft account team for details.
+Enterprise Agreement customers must purchase a quantity of Multi-Geo licenses equal to or greater than 5% of their total eligible seats. Similarly, CSP partners must purchase and assign a quantity of Multi-Geo licenses equal to or greater than 5% of their customer's total eligible Microsoft 365 seats. For Enterprise customers, user subscription licenses must be on the same Enterprise Agreement as the Multi-Geo Services licenses. Contact your Microsoft account team for details.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-multi-geo.md]:
```suggestion
Enterprise Agreement customers must purchase a quantity of Multi-Geo licenses equal to or greater than 5% of their total eligible seats. Similarly, CSP partners must purchase and assign a quantity of Multi-Geo licenses equal to or greater than 5% of their customers' total eligible Microsoft 365 seats. For Enterprise customers, user subscription licenses must be on the same Enterprise Agreement as the Multi-Geo Services licenses. Contact your Microsoft account team for details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MikePlumleyMSFT** [comment]
@AnupamFrancis can you review?

**Comment by AnfraMsft** [comment]
License assignment is expected for all satellite users in a Multi Geo tenant, same as assignment is expected for other products in general. Was there a specific reason this explicit mention is required?

@MikePlumleyMSFT BTW, I got locked out of my earlier account. This is my new one, for future reference :)

**Pull request merged**